src/lib/DefaultRuleManager.ts

```diff
@@ -107,8 +114,25 @@ export class DefaultRuleManager implements RuleManager {
 				this.logger.trace(`${e.getName()} is not eligible to execute this time.`);
 			}
 
+			if (paths.has(e.getName())) {
+				for (const t of engineTargets) {
+					for (const p of t.paths) {
+						paths.get(e.getName()).push(p);
+					}
+				}
+			}
 		}
 
+		// Checking if any file paths were processed by eslint and eslint-lwc, which may cause duplicate violations.
+		const pathsDoubleProcessed = paths.get(ENGINE.ESLINT).filter(path => paths.get(ENGINE.ESLINT_LWC).includes(path));
+		if (pathsDoubleProcessed.length === 1) {
+			uxEvents.emit(EVENTS.WARNING_ALWAYS, messages.getMessage('warning.pathDoubleProcessed'));
+		} else if (pathsDoubleProcessed.length > 1) {
+			uxEvents.emit(EVENTS.WARNING_ALWAYS, messages.getMessage('warning.pathsDoubleProcessed'));
+		}
```

Comment:
@gkjung Could you help understand - I feel like I'm missing something obvious - if the warning messages are going to be the same, why do we need an if/else? Can't we do a >=1 check?

Also, can we include the file names that got analyzed by both engines? A comma-separated string with contents of `pathsDoubleProcessed` would help users create a cleaner target pattern.

Reply:
Ah, got it, singular vs plural.
If we were to include the file names, would it be possible to keep it a single message?

Reply:
That would be cleaner, I'm not sure the best way to phrase the message to suit both. What if the first sentence was `At least one file was processed by both ESLint and ESLint-LWC, which may result in duplicate violations.`?

Reply:
I thought about including the file names, but if they had several I wasn't sure how to do that without the message getting really long. Any ideas?

Reply:
Good point. How about we put a limit on number of files included in the message? We could list out up to 10 file paths/names and conditionally add `and more` if the list has more remaining that we didn't cover.

I like the sentence structure. @teresa-allen-sfdc Please let us know your thoughts.

Reply:
This is what it's currently looking like `WARNING: At least one file was processed by both ESLint and ESLint-LWC simultaneously, which could result in duplicate violations. Customize targetPatterns properties for eslint and eslint-lwc engines in /Users/grace.jung/.sfdx-scanner/Config-pilot.json to remove overlap on the following file(s): /Users/grace.jung/Desktop/sfdx-scanner/test/code-fixtures/invalid-lwc/invalidApiDecorator/noLeadingUpperCase.js, /Users/grace.jung/Desktop/sfdx-scanner/test/code-fixtures/projects/dep-test-app/folder-a/SomeGenericFile.js, /Users/grace.jung/Desktop/sfdx-scanner/test/code-fixtures/projects/dep-test-app/folder-a/jquery-3.1.0.js, /Users/grace.jung/Desktop/sfdx-scanner/test/code-fixtures/projects/dep-test-app/folder-b/AnotherGenericFile.js, /Users/grace.jung/Desktop/sfdx-scanner/test/code-fixtures/projects/dep-test-app/folder-b/jquery-3.5.1.js, /Users/grace.jung/Desktop/sfdx-scanner/test/code-fixtures/projects/dep-test-app/folder-c/Burrito.js, and 26 more`. @teresa-allen-sfdc @rmohan20 How does this look? It only require one message for plural/singular and I listed out 5 because it was getting really long with 10

Reply:
@gkjung This looks great! I especially like the `and 26 more`. We could even stick to just two or three to drive the point across.

Reply:
I think three makes sense, and if they want to see the rest they can always fix those and rerun if there's any still overlapping.